zulipterminal/core.py

```diff
@@ -115,9 +115,9 @@ def show_msg_info(self, msg: Any) -> None:
         self.show_pop_up(msg_info_view,
                          "Message Information (up/down scrolls)")
 
-    def show_stream_info(self, color: Any, name: Any, desc: Any) -> None:
+    def show_stream_info(self, color: str, name: str, desc: str) -> None:
         show_stream_view = StreamInfoView(self, color, name, desc)
-        self.show_pop_up(show_stream_view, "Stream Description")
+        self.show_pop_up(show_stream_view, "Stream - {}".format(name))
```

Comment:
I'm wondering if the title might be cleaner just using eg `# announce` or `P some private stream`, as they are shown in the stream list, even without the color (color doesn't work?)

Reply:
I tried adding the color to the title of the popup. But I don't think it is possible with the curernt definition of the pop_up() function. Please correct me if I am wrong. And guide me on how to do it.

Reply:
I've given the tiltle a minimal look though.